components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__measurements.xml

```diff
@@ -41,8 +41,299 @@
+    </property>
+    <property>
+      <code>#measurements .list-actions {
+  width: auto;
+  clear: both;
+}
+#charts-new {
+  padding: 15px; padding-left: 190px;
+  height: 100%;
+  box-sizing: border-box;
+  float: right; width: 48%;
+  position: relative;
+}
+#charts-new .sets {
+  width: 120px;
+  position: absolute; top: 0; left: 0;
+}
+#charts-new .sets &gt; div {
+  cursor: pointer;
+  padding: 10px;
+}
+#charts-new .sets &gt; div.selected::after {
+  content: ' &gt;';
+}
+#charts-new .chart-thumbs {
+  width: 180px;
+  padding: 20px;
+  position: absolute; top: 0; left: 0;
+  overflow-y: scroll; overflow-x: hidden;
+  height: 100%;
+  box-sizing: border-box;
+  background-color: #FFFFFF;
+  border: 1px solid #FFFFFF;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px #DDDDDD inset;
+}
+#charts-new .chart-thumbs img {
+  width: 100px;
+  display: block;
+  margin: 0 auto;
+}
+#charts-new .chart-thumbs &gt; div {
+  border: 1px solid #aaa;
+  margin-bottom: 20px;
+  cursor: pointer;
+  background-color: white;
+  padding: 10px;
+}
+#charts-new .chart-thumbs &gt; div &gt; span {
+  text-align: center;
+  display: block;
+  padding-bottom: 5px;
+}
+#charts-new .chart-thumbs &gt; div.selected &gt; span {
+  font-weight: bold;
+}
+#charts-new .chart-thumbs &gt; div.selected {
+  border-color: #666;
+}
+#charts-new img.main {
+  max-width: 100%;
+  max-height: 100%;
+  padding: 0;
+}
+#measurements-container {
+  position: relative;
+  clear: both;
+}
+#measurements {
+  width: 485px;
+  float: left;
+}
+.measurement {
+  padding: 0.2em 0;
+  clear: both;
+  overflow: auto;
+}
+.measurement &gt; * {
+  float: left;
+}
+.measurement-set .delete {
+  float: right;
+  margin-left: 15px;
+}
+.measurement-row {
+  clear: both;
+}
+.measurement-row &gt; label,
+.measurement-row &gt; span,
+.measurement-row &gt; div,
+.measurement-row span.val
+ {
+  display: block;
+  float: left;
+  min-height: 1px;
+  line-height: 20px;
+}
+.measurement-row input {
+  width: auto; margin: 0 3px;
+} 
+.measurement-row span.val {
+  margin: 0 6px;
+  width: 40px;
+  text-align: right;
+}
+.measurement-row span.val.empty {
+  text-align: center;
+}
+.measurement-row div.metric,
+.measurement-row div.imperial {
+  width: 120px;
+}
+.viewbody .measurement-set label {
+  width: auto;
+}
+.measurement-row span.side-label {
+  width: 15px;
+  font-size: 80%;
+  font-weight: bold;
+  text-align: right;
+}
+.measurement-row span.unit {
+  width: 60px;
+}
+.measurement-row span.pctl {
+  width: 120px;
+  font-size: 80%;
+}
+.editbody .measurement-row &gt; label,
+.viewbody .measurement-row &gt; label {
+  width: 180px;
+}
+.normal { 
+  color: green; 
+}
+.below-normal,
+.above-normal {
+  color: orange;
+}
+.extreme-below-normal,
+.extreme-above-normal {
+  color: red;
+}
+.measurement-set.proto {
+  display: none;
+}
+.measurement-set {
+  border: 1px dotted #aaa;
+  padding: 10px;
+  float: left;
+  clear: both;
+  overflow: auto;
+  margin: 5px;
+  position: relative;
```

Comment:
Don't mean to be picking on you, since this is probably something we do everywhere, but the ordering of the CSS properties is kind of important. We should consider following something like this: https://github.com/necolas/idiomatic-css#declaration-order